### ISSUE TITLE
AttackHi32

### ISSUE THREAD

**Issue by MalustheEnigma** [Project-M-CC/lyn#11]
I've noticed that the hitboxes on this move are seemingly nonexistent, I've had enemies walk right through the sword being whirled around in the air. I'd attempt to fix this myself, but BrawlBox won't let me open Lyn's .pac file in the previewer and PSA doesn't recognize that Lyn has an AttackHi32 move. While I'm at it, does anybody know how I could open Lyn's .pac file in the previewer?

**Comment by TheSmashBoy** [comment]
Her second utilt should be recognizeable in the PSA. You just need to add hitboxes during those frames where her sword spins. I can fix that actually. But now, I wanna update my post which is important regarding her completion.

**Comment by SzionBlue** [comment]
did that fix ever happen?

**Comment by TRASH129** [comment]
@MalustheEnigma 
>hitboxes on this move are seemingly nonexistent

You are correct. As of December 7, 2016 there are absolutely **no hitboxes on the spinning part** of "AttackHi32". It's only until she swings her sword down when hitboxes appear.

> PSA doesn't recognize that Lyn has an AttackHi32 move

I would like to mention that **"AttackHi32" is located in subaction "4D" while "AttackHi" is located in subaction "54"** in PSA. In BrawlBox, the animation pane has "AttackHi32" right below "AttackHi" but **in the moveset pane, "AttackHi32" is located 7-ish slots above "AttackHi"**.

> BrawlBox won't let me open Lyn's .pac file in the previewer
> does anybody know how I could open Lyn's .pac file in the previewer?

I am going to assume you mean the model previewer, and I shall give you some anecdotal advice. Until recently, I had a laptop with Windows XP Service Pack 3 along with a grand total of 512 megabytes of RAM. I had BrawlBox .68b installed and I had a problems with seemingly random .pac and .pcs files crashing BrawlBox when I tried to view the file with the model previewer. Being the superstitious teenager I was, I did a bunch of rituals to help alleviate the problem. I would **open the file slowly** while waiting 15 seconds before moving my mouse or pressing any key, view the .pcs file instead of the .pac file, open and close the .pac file multiple times, **move the .pac file to a different folder**, navigate only using the keyboard, re download the .pac file, or I would **open and highlight a couple of the model's branch folders/files without highlighting the mdl0 file itself before closing the mdl0 branch and clicking the mdl0 file to actually preview the model**. I would try all these things and more until it started working again (I swear that last one did the trick almost every single time). Thankfully, I was gifted a Windows 10 with 8 gigabytes of RAM and I never encountered this issue again. Who knows, maybe one of my superstitions was right for the wrong reasons...
TL;DR
I have no idea how but try doing some of the bold parts and see what happens.

P.S. Please don't ban me if necroposting is a thing on github. I haven't read Da Rules yet.

**Comment by TheSmashBoy** [comment]
You people are complicating your lives. First of all, Lyn is no BrawlBox PSA she is a pure PSA make up. You use PSA to open her up and simply add those hitboxes to the proper frames. Even I did this and it was super easy.

Also, the ordering of moves in PSA is super insignificant. Take her AttackLw3; it's at the bottom of her moveset list in her BrawlBox preview yet it still works just as her Hi32, but anyways; you use BrawlBox to look at which frames the sword starts spinning so you can pinpoint on what frame the Asyncronous Timer can start or end or just put AT and use Synchronous Timers to set how long the hitboxes last during the spin.

**Comment by MalustheEnigma** [comment]
Thank you for the information.

**Comment by MalustheEnigma** [comment]
Thank you for the information.

**Comment by TheSmashBoy** [comment]
@MalustheEnigma No problem.


Also, I made some updates to Lyn, and one of them is her shared soundbank with Ike, the famous Ike+Lyn soundbank Purpa made but never published. You can grab it at the Lyn server I made in the updates-files-log channel. For those who haven't joined in yet.
